util/EIP-152/benches/bench.rs

```diff
@@ -0,0 +1,179 @@
+            000000000000000000000000000000000000000000000000000000000000000000000000000000000000000\
+            000000000000000000000000000000000000000000000000000000000000000000000000000000000000000\
+            0000000000000000000000000000000000000000000000300000000000000000000000000000001",
+                        "08c9bcf367e6096a3ba7ca8485ae67bb2bf894fe72f36e3cf1361d5f3af54fa5d282e6ad7f520e511f6c3e\
+            2b8c68059b9442be0454267ce079217e1319cde05b",
+                    ).to_parts(),
+                    TestVector::from_string(
+                        "0000000c48c9bdf267e6096a3ba7ca8485ae67bb2bf894fe72f36e3cf1361d5f3af54fa5d182e6ad7f520e\
+            511f6c3e2b8c68059b6bbd41fbabd9831f79217e1319cde05b6162630000000000000000000000000000000\
+            000000000000000000000000000000000000000000000000000000000000000000000000000000000000000\
+            000000000000000000000000000000000000000000000000000000000000000000000000000000000000000\
+            0000000000000000000000000000000000000000000000300000000000000000000000000000001",
+                        "ba80a53f981c4d0d6a2797b69f12f6e94c212f14685ac4b74b12bb6fdbffa2d17d87c5392aab792dc252d5\
+            de4533cc9518d38aa8dbf1925ab92386edd4009923",
+                    ).to_parts(),
+                    TestVector::from_string(
+                        "0000000c48c9bdf267e6096a3ba7ca8485ae67bb2bf894fe72f36e3cf1361d5f3af54fa5d182e6ad7f520e\
+            511f6c3e2b8c68059b6bbd41fbabd9831f79217e1319cde05b6162630000000000000000000000000000000\
+            000000000000000000000000000000000000000000000000000000000000000000000000000000000000000\
+            000000000000000000000000000000000000000000000000000000000000000000000000000000000000000\
+            0000000000000000000000000000000000000000000000300000000000000000000000000000000",
+                        "75ab69d3190a562c51aef8d88f1c2775876944407270c42c9844252c26d2875298743e7f6d5ea2f2d3e8d2\
+            26039cd31b4e426ac4f2d3d666a610c2116fde4735",
+                    ).to_parts(),
+                    TestVector::from_string(
+                        "0000000148c9bdf267e6096a3ba7ca8485ae67bb2bf894fe72f36e3cf1361d5f3af54fa5d182e6ad7f520e\
+            511f6c3e2b8c68059b6bbd41fbabd9831f79217e1319cde05b6162630000000000000000000000000000000\
+            000000000000000000000000000000000000000000000000000000000000000000000000000000000000000\
+            000000000000000000000000000000000000000000000000000000000000000000000000000000000000000\
+            0000000000000000000000000000000000000000000000300000000000000000000000000000001",
+                        "b63a380cb2897d521994a85234ee2c181b5f844d2c624c002677e9703449d2fba551b3a8333bcdf5f2f7e0\
+            8993d53923de3d64fcc68c034e717b9293fed7a421",
+                    ).to_parts()
+                ];
+
+                for (mut state, message, count, f, rounds, expected) in vectors {
```

Comment:
there is too much noise in what we're measuring
let's make it a benchmark with constant input from `test_blake2_f` except for number of rounds
(https://bheisler.github.io/criterion.rs/book/user_guide/benchmarking_with_inputs.html#benchmarking-with-a-range-of-values)
not need for `assert_eq!`, we're not trying to find bugs here, adding `black_box` if necessary to avoid the compiler to optimize the call away